### PR TITLE
MC-13922: Added doc for retrieving waf settings for a site

### PR DESCRIPTION
**Pull request by RGitB**
#### Changes made

- Added docs for retrieving WAF settings from API

### CODE REVIEW

File: source/includes/stackpath/_sites.md [hunk 1]
```diff
@@ -365,4 +365,119 @@ Disable Serverless Scripts for a given site.
...
+```json
+{
+   "data":{
+      "siteId":"1b1cd7e6-41ab-4e0f-a59a-5c4b89da1b36",
+      "stackId":"1415650d-1d02-4097-a79a-8f6e4bb4f483",
+      "apiUrls":[
+         "/test/this/way"
+      ],
+      "ddosSetting":{
+         "globalThreshold":"5000",
```
Comment (patrickspensieri) [inline source/includes/stackpath/_sites.md]: Should have spaces between labels and values

File: source/includes/stackpath/_sites.md [hunk 2]
```diff
@@ -365,4 +365,119 @@ Disable Serverless Scripts for a given site.
...
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/wafsettings/<a href="#administration-sites">:site_id</a></code>
+
+Retrieve the WAF Settings for a site.
+
+Attributes | &nbsp;
+------- | -----------
+`siteId`<br/>*string* | The ID of the site that the WAF settings belongs to.
+`stackId`<br/>*string* | The ID of the stack that a site belongs to.
+`ddosSetting`<br/>*object* | The DDOS Setting containing the different threshold values.
```
Comment (patrickspensieri) [inline source/includes/stackpath/_sites.md]: Can we also document what the available DDoS settings are? For example (Also, should be written DDoS)
```
 `ddosSettings.globalThreshold`<br/>*Integer* | ...
```

File: source/includes/stackpath/_sites.md [hunk 3]
```diff
@@ -365,4 +365,119 @@ Disable Serverless Scripts for a given site.
...
+`apiUrls`<br/>*array* | List of configured API urls.
+`isMonitoring`<br/>*boolean* | If the monitoring mode is enabled.
+`listPolicyGroups`<br/>*object* | An object containing the list of Policy Groups.
+`policyGroups`<br/>*array[object]* | A list of policy group object which contains id, name and list of policies.
+`policies`<br/>*array[object]* | A list of policies which contains id, name, description, action and enabled.
```
Comment (patrickspensieri) [inline source/includes/stackpath/_sites.md]: Same here, should also document what each field is and some detail on each, like what the available actions are.

File: source/includes/stackpath/_sites.md (hunk 1 above)
Comment (RGitB) [reply source/includes/stackpath/_sites.md]: my json formatter messed it up. Thanks, I'll update it  494d76f

File: source/includes/stackpath/_sites.md (hunk 3 above)
Comment (RGitB) [reply source/includes/stackpath/_sites.md]: 494d76f

File: source/includes/stackpath/_sites.md (hunk 2 above)
Comment (RGitB) [reply source/includes/stackpath/_sites.md]: 494d76f

**Review by patrickspensieri — APPROVED** [review]
Thanks for making the changes

**Review by evanluc — APPROVED** [review]

**Pull request merged**